exercises/leap/leap_tests.plt

```diff
@@ -1,19 +1,24 @@
-
 :- begin_tests(leap_tests).
-:- use_module(library(plunit)).
 
-    test(leap_year):- leap(1996).
+    test(leap_year) :-
```

Comment:
This `:-` is called the `neck` symbol. Most prolog code I have seen has a space between the head and the neck. Also, the "body" should always be on a new line I think.